mass_mailing_unique/models/mail_mass_mailing_list.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class MailMassMailingList(models.Model):
+    _inherit = 'mail.mass_mailing.list'
+
+    @api.constrains('contact_ids')
+    def _check_contact_ids_email(self):
+        contact_obj = self.env['mail.mass_mailing.contact']
```

Comment:
These 3 constraints do the same thing. Can't we fall back to just one?

For instance, I think the most readable one is the one found in `mail.mass_mailing.contact`. Then, here you could just do:

```python
self.mapped("contact_ids")._check_email_list_ids()
```